architecture/decision/records/002-use-products-repo.md

```diff
@@ -0,0 +1,26 @@
+# ADR-002 Use a dedicated repository for data product defintions
+
+## Status
+
+🤔 Proposed
```

Comment:
This has been decided now

Reply:
Fair enough. In that case I'll also add ADR-002 to the index.